src/libs/elektra/keyvalue.c

```diff
@@ -353,6 +362,10 @@ ssize_t keyGetString (const Key * key, char * returnedString, size_t maxSize)
  * regardless of the program's current encoding (if the iconv plugin is
  * available).
  *
+ * @pre @p newStringValue is a NULL terminated string
+ * @post Value of the Key is set to the UTF-8 encoded value of @p newStringValue
+ * @post Metadata keys "binary" / "origvalue" are cleared
```

Comment:
origvalue is a hack with a wrong name (it should be `meta:/original/value` or similar), we shouldn't have this as part of our API.

See #3626 for more information about origvalue.

```suggestion
 * @post Metakey `meta:/binary` is cleared
```